File: VB/GridLookUpEdit_MultiAutoSearch/Form1.vb
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms

Namespace GridLookUpEdit_MultiAutoSearch
    Partial Public Class Form1
        Inherits Form

        Public Sub New()
            InitializeComponent()
        End Sub

        Private Sub Form1_Load(ByVal sender As Object, ByVal e As EventArgs) Handles Me.Load
            ' TODO: This line of code loads data into the 'nwindDataSet.Customers' table. You can move, or remove it, as needed.
            Me.customersTableAdapter.Fill(Me.nwindDataSet.Customers)

        End Sub

        Private Sub gridLookUpEdit1_AutoSearch(sender As Object, e As DevExpress.XtraEditors.Controls.LookUpEditAutoSearchEventArgs) Handles gridLookUpEdit1.AutoSearch
            e.SetParameters(DevExpress.Data.Filtering.FindPanelParserKind.Exact, DevExpress.Data.Filtering.FilterCondition.Contains)
        End Sub
    End Class
End Namespace

File: CS/GridLookUpEdit_MultiAutoSearch/Form1.cs
using DevExpress.XtraEditors;
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;

namespace GridLookUpEdit_MultiAutoSearch {
    public partial class Form1 : XtraForm {
        public Form1() {
            InitializeComponent();
        }

        private void Form1_Load(object sender, EventArgs e) {
            // TODO: This line of code loads data into the 'nwindDataSet.Customers' table. You can move, or remove it, as needed.
            this.customersTableAdapter.Fill(this.nwindDataSet.Customers);
        }

        private void gridLookUpEdit1_AutoSearch(object sender, DevExpress.XtraEditors.Controls.LookUpEditAutoSearchEventArgs e) {
            e.SetParameters(DevExpress.Data.Filtering.FindPanelParserKind.Exact, DevExpress.Data.Filtering.FilterCondition.Contains);
        }
    }
}